dashboard/test/factories/factories.rb

```diff
@@ -701,6 +705,20 @@
     sequence(:name) {|n| "bogus-script-#{n}"}
     published_state "beta"
 
+    trait :with_levels do
+      ignore do
+        levels_count 0
+      end
```

Comment:
I'm having trouble finding the docs for `ignore`. can you point me to where you found out about this method?

Reply:
oh, right! I grabbed that from https://github.com/thoughtbot/factory_bot/issues/527, which includes a note that `ignore` should be replaced with `transient`, which I totally forgot to do